homematic/data/run.sh

```diff
@@ -32,16 +25,16 @@ touch /data/homematic.regadom
 . /usr/lib/hm-interface.sh
 
 # Setup Interfaces
-init_interface_list "$RF_ENABLE" "$HMIP_ENABLE" "$WIRED_ENABLE"
+init_interface_list "$(bashio::config 'rf_enable')" "$(bashio::config 'hmip_enable')" "$(bashio::config 'wired_enable')"
```

Comment:
Lets make this line shorter for readability 
```suggestion
init_interface_list \
    "$(bashio::config 'rf_enable')" \
    "$(bashio::config 'hmip_enable')" \
    "$(bashio::config 'wired_enable')"
```